### ISSUE TITLE
Bundle not compatiple with Symfony 3.0

### ISSUE THREAD

**Issue by remoteclient** [sonata-project/SonataTranslationBundle#64]
I wanted to upgrade symfony to new major version 3.0. composer gives me this error:

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - sonata-project/translation-bundle 1.0.x-dev requires symfony/symfony ~2.2 -> no matching package found.
    - sonata-project/translation-bundle 1.0.x-dev requires symfony/symfony ~2.2 -> no matching package found.
    - sonata-project/translation-bundle 1.0.x-dev requires symfony/symfony ~2.2 -> no matching package found.
    - Installation request for sonata-project/translation-bundle == 1.0.9999999.9999999-dev -> satisfiable by sonata-project/translation-bundle[1.0.x-dev].
```

**Comment by andrewtch** [comment]
Same thing here. Maybe let's jsut adjust composer requirement to ~2.2|~3.0?

**Comment by core23** [comment]
Should be fixed with https://github.com/sonata-project/SonataTranslationBundle/pull/71

**Comment by dbu** [comment]
fixed in #71
